platform/rct/Project/STM32_FLASH.ld

```diff
@@ -42,17 +42,19 @@ _Min_Stack_Size = 0x400; 	/* required amount of stack */
 /* Specify the memory areas */
 MEMORY
 {
-  FLASH     (rx)    : ORIGIN = 0x08000000, LENGTH = 256K
-  
+  BOOTLDR   (rx)    : ORIGIN = 0x08000000, LENGTH = 16K
+  FLASH     (rx)    : ORIGIN = 0x08004000, LENGTH = 256K - 48K
+
   CONFIG    (rx)    : ORIGIN = 0x08038000, LENGTH = 16K
```

Comment:
Probably don't need the 'x' flag set on either CONFIG or TRACKS wouldn't you agree?  Could be done in a separate PR too